frontend/src/app/main/ui/viewer/handoff/attributes/text.cljs

```diff
@@ -0,0 +1,164 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020 UXBOX Labs SL
+
+(ns app.main.ui.viewer.handoff.attributes.text
+  (:require
+   [rumext.alpha :as mf]
+   [cuerdas.core :as str]
+   [app.util.data :as d]
+   [app.util.i18n :refer [t]]
+   [app.util.color :as uc]
+   [app.util.text :as ut]
+   [app.main.fonts :as fonts]
+   [app.main.ui.icons :as i]
+   [app.util.webapi :as wapi]
+   [app.main.ui.viewer.handoff.attributes.common :refer [copy-cb color-row]]))
+
+(defn has-text? [shape]
+  (:content shape))
+
+(def properties [:fill-color
+                 :fill-color-gradient
+                 :font-family
+                 :font-style
+                 :font-size
+                 :line-height
+                 :letter-spacing
+                 :text-decoration
+                 :text-transform])
+
+(defn shape->color [shape]
+  {:color (:fill-color shape)
+   :opacity (:fill-opacity shape)
+   :gradient (:fill-color-gradient shape)
+   :id (:fill-ref-id shape)
+   :file-id (:fill-ref-file-id shape)})
+
+(defn format-style [color]
+  {:font-family #(str "'" % "'")
+   :font-style #(str "'" % "'")
+   :font-size #(str % "px")
+   :line-height #(str % "px")
+   :letter-spacing #(str % "px")
+   :text-decoration name
+   :text-transform name
+   :fill-color #(uc/color->background color)
+   :fill-color-gradient #(uc/color->background color)})
+
+(mf/defc typography-block [{:keys [shape locale text style full-style]}]
+  (let [color-format (mf/use-state :hex)
+        color (shape->color style)
+        to-prop {:fill-color "color"
+                 :fill-color-gradient "color"}]
+    [:div.attributes-text-block
+     [:div.attributes-typography-row
+      [:div.typography-sample
+       {:style {:font-family (:font-family full-style)
+                :font-weight (:font-weight full-style)
+                :font-style (:font-style full-style)}}
+       (t locale "workspace.assets.typography.sample")]
```

Comment:
I think this text is not in figma prototype. And in locales.json is just "Ag" ¿?

Reply:
Yes it is